### PR TITLE
Add nexus recipe

### PR DESCRIPTION
**Pull request by beenje**
I don't know much about windows, so I just skip it for now.
Not tested on OSX.

### CODE REVIEW

**Comment by beenje** [comment]
Hi @conda-forge/core,

This is a dependency for [Mantid](http://www.mantidproject.org/Main_Page) that I'd like added to conda-forge. Anyone has the time to give some feedback?
Thanks!

File: recipes/nexus/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    - hdf4
+    - jpeg
+    - libxml2
+    - ncurses
+    - readline
+    - zlib
+  run:
+    - hdf5
+    - hdf4
+    - jpeg
```
Comment (hmaarrfk) [inline recipes/nexus/meta.yaml]: no need for jpeg. it includes a run exports
https://github.com/conda-forge/jpeg-feedstock/blob/master/recipe/meta.yaml#L15

File: recipes/nexus/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    - libxml2
```
Comment (hmaarrfk) [inline recipes/nexus/meta.yaml]: no need for libxml2 here too
https://github.com/conda-forge/libxml2-feedstock/blob/master/recipe/meta.yaml#L15

File: recipes/nexus/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    - ncurses
+    - readline
```
Comment (hmaarrfk) [inline recipes/nexus/meta.yaml]: readline also includes run exports
https://github.com/conda-forge/readline-feedstock/blob/master/recipe/meta.yaml#L15

File: recipes/nexus/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    - zlib
```
Comment (hmaarrfk) [inline recipes/nexus/meta.yaml]: Same for zlib
https://github.com/conda-forge/zlib-feedstock/blob/master/recipe/meta.yaml#L15

**Comment by beenje** [comment]
I removed the unnecessary run requirements.

I skipped windows by lack of knowledge and time...
I'd be glad if someone wanted to help.
Those instructions don't sound straightforward to me: https://github.com/nexusformat/code#building-the-code-on-windows
I honestly don't have the time to investigate windows build right now...

**Comment by chrisburr** [comment]
I added `make` for completeness and I think Windows can be added in the feedstock if someone wants it.

Thanks for the contribution @beenje and welcome to the team! 👍

**Comment by beenje** [comment]
Thanks!

**Pull request merged**